packages/stream_chat_flutter/lib/src/message_input/stream_message_input.dart

```diff
@@ -614,6 +749,12 @@ class StreamMessageInputState extends State<StreamMessageInput>
     );
   }
 
+  Widget _buildConfirmRecordButton() {
+    return OnPressButton.confirmAudio(onPressed: () {
+      _finishRecording(context);
+    });
+  }
```

Comment:
Can we inline some of these? Having methods which return Widgets might lower performance.